### ISSUE TITLE
Make playlist resizeable (change width)

### ISSUE THREAD

**Issue by stefansimik** [iina/iina#4887]
**What you want IINA to do:**

Please allow for resizable playlist, so we can make it wider and see full name of all files.

**What IINA does currently:**

Currently it has fixed width, which results, that only a few first + last characters are visible 
and you cannot see the full name of files with longer names.

**Comment by low-batt** [comment]
Rather than a feature request I am going to label this as an Apple Human Interface Guidelines violation. Because IINA does not change the cursor when positioned close to the playlist users are not aware that the playlist can be resized as made clear by this report. IINA should be displaying the resize cursors listed in the [pointers](https://developer.apple.com/design/human-interface-guidelines/pointing-devices#Pointers) section of the HIG.

There is already PR #4472 that attempts to address this. From what I recall it is currently a draft due to problems getting the cursor change to work properly. I believe this PR also increases the limit on the size the playlist is allowed to grow.

If you hover over the `…` in a compressed filename the playlist will show the full filename for that entry. Of course this is not as useful as being able to increase the size of the playlist so filenames are not compressed.

**Comment by stefansimik** [comment]
> Rather than a feature request I am going to label this as an Apple Human Interface Guidelines violation. Because IINA does not change the cursor when positioned close to the playlist users are not aware that the playlist can be resized as made clear by this report. IINA should be displaying the resize cursors listed in the [pointers](https://developer.apple.com/design/human-interface-guidelines/pointing-devices#Pointers) section of the HIG.
> 
> There is already PR #4472 that attempts to address this. From what I recall it is currently a draft due to problems getting the cursor change to work properly. I believe this PR also increases the limit on the size the playlist is allowed to grow.
> 
> If you hover over the `…` in a compressed filename the playlist will show the full filename for that entry. Of course this is not as useful as being able to increase the size of the playlist so filenames are not compressed.

That's right. I would just stress more one thing - **after a lot of effort**, I was is able to resize the playlist - the ability to change width is extremely limited and there is maximum width, that is almost the same like initial width - as a result, it does not help much at all.

It is really important to be able to expand playlist into separate window, that is completely resizeable as we need.
The docked version is OK, but for any bigger playlist - it is very uncomfortable and impractical design of UI.
There is a strong need to be able to move playlist into 100% resizeable separate window like VLC has.

**Comment by low-batt** [comment]
Yes, for the longest time I too was unaware the playlist could be resized. The intent of PR #4472 is to bring attention to the user that the playlist can be resized and raise the size limit.

On this:
> It is really important to be able to expand playlist into separate window, that is completely resizeable as we need.

See issue #42. I'm a junior IINA developer. I do not have a background in UI design and coding, so I will defer to the senior developers on the issue of a separate window for the playlist. There are requests for being able to move the on screen controller out of the window as well.

**Comment by low-batt** [comment]
This was fixed in IINA 1.4.0.
